Tests/NIOSSLTests/SSLPrivateKeyTests.swift

```diff
@@ -237,17 +237,6 @@ class SSLPrivateKeyTest: XCTestCase {
         }
     }
 
-    func testMissingPassword() {
```

Comment:
We can still test this path (the functionality isn't going away), but we'll need to make the tests as deprecated too (they'll still be run).